websites/T/Trello/dist/metadata.json

```diff
@@ -14,9 +14,9 @@
     "blog.trello.com",
     "developers.trello.com"
   ],
-  "version": "2.1.9",
-  "logo": "https://i.imgur.com/5IK6wjd.png",
-  "thumbnail": "https://i.imgur.com/6V8VR6S.png",
+  "version": "2.1.91",
```

Comment:
tf is that version

Reply:
Dad to bump it, but didn't want to make it 2.2 since I didn't edit the code. Should I?